Sources/TunnelKitWireGuardAppExtension/WireGuardTunnelProvider.swift

```diff
@@ -146,6 +176,27 @@ extension WireGuardTunnelProvider {
         // store path for clients
         cfg._appexSetDebugLogPath()
     }
+
+    func fetchDataCount(completiondHandler: @escaping (Result<WireGuardDataCount, Error>) -> Void) {
+        adapter.getRuntimeConfiguration { configurationString in
+            if let wireGuardDataCount = WireGuardDataCount(from: configurationString) {
```

Comment:
According to the other suggestion:

```suggestion
            if let configurationString, let wireGuardDataCount = WireGuardDataCount(from: configurationString) {
```

Reply:
yep!